### DISCUSSION TITLE
Command Palette Won't Appear

### DISCUSSION THREAD

**Discussion by MrRogerSmith** [atom/atom#23527] [Q&A] [answered]
I'm pretty new to Atom.  The instructions say to activate the command palette by pressing ctrl+shift+P.  I've tried that repeatedly and nothing happens.  Is there a setting that could be blocking the command palette from showing up?  Or any other ideas?

**Comment by icecream17** [comment, marked as answer]
What happens when you click Command Palette in the menu?

![View > Toggle Command Palette](https://user-images.githubusercontent.com/58114641/151626515-07332167-db34-46ad-acef-2eb1cff04483.png)

Also see https://flight-manual.atom.io/hacking-atom/sections/debugging/#check-your-keybindings (what happens when you try the combination in the keybinding resolver?)

**Reply by MrRogerSmith** [reply]
Thank you for your response.

I tried your suggestion to check the Keybinding Resolver.  Sure enough, it shows that when I press control + shift + P, it's only registering the control + shift.  And this is the only combo this happens for.  Control + P works, Shift + P works, and control + shift + any other letter works.  So then I tried that key combo in some other programs and it's also not registering there.  So I guess it's a problem with my own system.

Thanks for your help.
